samples/csharp/getting-started/Regression_AutoML/TaxiFarePrediction/Program.cs

```diff
@@ -0,0 +1,329 @@
+            Console.WriteLine($"Predicted fare: {predictedResult.FareAmount:0.####}, actual fare: 15.5");
+            Console.WriteLine($"**********************************************************************");
+        }
+
+        private static void PlotRegressionChart(MLContext mlContext,                                               
+                                                string testDataSetPath,
+                                                int numberOfRecordsToRead,
+                                                string[] args)
+        {
+            ITransformer trainedModel;
+            using (var stream = new FileStream(ModelPath, FileMode.Open, FileAccess.Read, FileShare.Read))
+            {
+                trainedModel = mlContext.Model.Load(stream, out var modelInputSchema);
+            }
+
+            // Create prediction engine related to the loaded trained model
+            var predFunction = mlContext.Model.CreatePredictionEngine<TaxiTrip, TaxiTripFarePrediction>(trainedModel);
+
+            string chartFileName = "";
+            using (var pl = new PLStream())
+            {
+                // use SVG backend and write to SineWaves.svg in current directory
+                if (args.Length == 1 && args[0] == "svg")
+                {
+                    pl.sdev("svg");
+                    chartFileName = "TaxiRegressionDistribution.svg";
+                    pl.sfnam(chartFileName);
+                }
+                else
+                {
+                    pl.sdev("pngcairo");
+                    chartFileName = "TaxiRegressionDistribution.png";
+                    pl.sfnam(chartFileName);
+                }
+
+                // use white background with black foreground
+                pl.spal0("cmap0_alternate.pal");
+
+                // Initialize plplot
+                pl.init();
+
+                // set axis limits
+                const int xMinLimit = 0;
+                const int xMaxLimit = 35; //Rides larger than $35 are not shown in the chart
+                const int yMinLimit = 0;
+                const int yMaxLimit = 35;  //Rides larger than $35 are not shown in the chart
+                pl.env(xMinLimit, xMaxLimit, yMinLimit, yMaxLimit, AxesScale.Independent, AxisBox.BoxTicksLabelsAxes);
+
+                // Set scaling for mail title text 125% size of default
+                pl.schr(0, 1.25);
+
+                // The main title
+                pl.lab("Measured", "Predicted", "Distribution of Taxi Fare Prediction");
+
+                // plot using different colors
+                // see http://plplot.sourceforge.net/examples.php?demo=02 for palette indices
+                pl.col0(1);
+
+                int totalNumber = numberOfRecordsToRead;
+                var testData = new TaxiTripCsvReader().GetDataFromCsv(testDataSetPath, totalNumber).ToList();
+
+                //This code is the symbol to paint
```

Comment:
```suggestion
                // This code is the symbol to paint
```

Reply:
Could use some comment clean up in the file (space after \\\\ & begin with capital letter)